CHANGELOG.md

```diff
@@ -13,6 +13,11 @@ A `secondaryLevel: true` option is available to add operations to the widget's c
 ### Fixes
 
 * Fixes all eslint warnings.
+* Fixes a RT bug where including `table` in `toolbar` but omitting an `insert` array crashed the rich text editor.
```

Comment:
This already shipped, see duplicate entry further down for 4.15.1

Reply:
Deleted.